privatePackages/assert-project/test/tsconfig.json

```diff
@@ -1,25 +1,11 @@
 {
+  "extends": "../../utils/tsconfig.json",
```

Comment:
this should be `"../../../utils/tsconfig.json"`

Reply:
Yeah, I&nbsp;used a&nbsp;script which had&nbsp;the&nbsp;`extends` path hard&#x2011;coded.